extendr-api/Cargo.toml

```diff
@@ -22,12 +22,14 @@ ndarray = { version = "0.15.3", optional = true }
 lazy_static = "1.4"
 paste = "1.0.5"
 serde = { version = "1.0", features = ["derive"], optional = true }
+num-complex = { version = "0.4", optional = true }
 
 [features]
-default = ["serde"]
+# change this to [] in master.
+default = ["num-complex"]
```

Comment:
Is this something that should be fixed before merging?

Reply:
Yes. I intend to write a simple alternative that does not use num-complex
as the default. I am mindful that every crate we add as a non-optional dependency
costs on build times.